code_samples/recent_activity/src/Command/ActivityLogContextTestCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Event\MyFeatureEvent;
+use App\MyFeature\MyFeature;
+use Ibexa\Contracts\ActivityLog\ActivityLogServiceInterface;
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\ContentTypeService;
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\LocationCreateStruct;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class ActivityLogContextTestCommand extends Command
+{
+    protected static $defaultName = 'doc:test:activity-log-context';
+
+    private ActivityLogServiceInterface $activityLogService;
+
+    private ContentService $contentService;
+
+    private ContentTypeService $contentTypeService;
+
+    private EventDispatcherInterface $eventDispatcher;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    public function __construct(ActivityLogServiceInterface $activityLogService, ContentService $contentService, ContentTypeService $contentTypeService, EventDispatcherInterface $eventDispatcher, PermissionResolver $permissionResolver, UserService $userService)
+    {
```

Comment:
To increase readability we usually split long lines in method declarations. Don't know if the same rules should be applied to suggested docs, but I think it should.

```suggestion
    public function __construct(
        ActivityLogServiceInterface $activityLogService,
        ContentService $contentService,
        ContentTypeService $contentTypeService,
        EventDispatcherInterface $eventDispatcher,
        PermissionResolver $permissionResolver,
        UserService $userService
    ) {
```
```